msteams-platform/m365-apps/publish.md

```diff
@@ -8,7 +8,7 @@ ms.localizationpriority: medium
 ---
 # Publish Teams apps for Microsoft 365
 
-Microsoft 365-enabled Teams apps are supported for production use in Microsoft Teams. You can distribute these apps to preview audiences who use the *Targeted Release* versions of outlook.com and office.com, and the *Beta Channel* build of Outlook for Windows desktop. Distribution options and processes for Microsoft 365-enabled Teams apps are the same as for traditional Teams apps.
+Microsoft 365-enabled Teams apps are supported for production use in Microsoft Teams. You can distribute these apps to preview audiences who use the *Targeted Release* versions of outlook.com and office.com, and the *Beta Channel* build of Outlook for Windows desktop, and Office Current Channel (Preview) build of Office app for Android. Distribution options and processes for Microsoft 365-enabled Teams apps are the same as for traditional Teams apps.
```

Comment:
Whoops, this needs updated from:

You can distribute these apps to preview audiences who use the *Targeted Release* versions of outlook.com and office.com, and the *Beta Channel* build of Outlook for Windows desktop, and Office Current Channel (Preview) build of Office app for Android.

To:

You can distribute these apps to preview audiences who use the *Targeted Release* versions of outlook.com and office.com, the *Beta Channel* build of Outlook for Windows desktop, and beta program builds of Office app for Android.

and